README.adoc

```diff
@@ -66,9 +66,7 @@ compiler currently runs on the following platforms:
 |                |  Tier 1 (actively maintained)   | Tier 2 (maintained when possible)
 
 | x86 64 bits    | Linux, macOS, Windows, FreeBSD  |  NetBSD, OpenBSD
-| x86 32 bits    | Linux, Windows                  |  FreeBSD, NetBSD, OpenBSD
 | ARM 64 bits    | Linux, macOS                    |  FreeBSD
-| ARM 32 bits    | Linux                           |  FreeBSD, NetBSD, OpenBSD
 | Power 64 bits  | Linux                           |
 | Power 32 bits  | Linux                           |
```

Comment:
As pointed out by @dra27, this line should be removed, as PPC32 is not coming back either.  (PPC should be back some day, but in 64-bit little-endian mode only.)

Reply:
Thanks, fixed.